File: FitnessCentar/UI/fitness_admin/lib/models/korisnici.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnici.g.dart';

@JsonSerializable()
class Korisnici{
int? id;
String? ime;
String? prezime;
String? korisnickoIme;
String? email;
String? telefon;
DateTime? datumRegistracije;
DateTime? datumRodjenja;
String? pol;
double? tezina;
double? visina;
String? slika;
String? lozinka;

  Korisnici(this.id,this.ime,this.prezime,this.korisnickoIme,this.email,this.telefon,this.datumRegistracije,this.datumRodjenja,this.pol,this.tezina,this.visina,this.slika,this.lozinka);




  factory Korisnici.fromJson(Map<String, dynamic> json) =>
      _$KorisniciFromJson(json);

  Map<String, dynamic> toJson() => _$KorisniciToJson(this);


}

File: FitnessCenatr.Model/Korisnici.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitnessCentar.Model
{
    public partial class Korisnici
    {
        public int Id { get; set; }

        public string? Ime { get; set; }

        public string? Prezime { get; set; }

        public string? KorisnickoIme { get; set; }

        // public string? Lozinka { get; set; }

        public string? Email { get; set; }

        public string? Telefon { get; set; }

        public DateTime? DatumRegistracije { get; set; }

        public DateTime? DatumRodjenja { get; set; }

        public string? Pol { get; set; }

        public decimal? Tezina { get; set; }

        public decimal? Visina { get; set; }

        public virtual Treneri Trener { get; set; } = null!;
        public byte[]? Slika { get; set; }

        //   public virtual ICollection<Administracija> Administracijas { get; set; } = new List<Administracija>();

        //  public virtual ICollection<Komentari> Komentaris { get; set; } = new List<Komentari>();

        // public virtual ICollection<Napredak> Napredaks { get; set; } = new List<Napredak>();

        // public virtual ICollection<Placanja> Placanjas { get; set; } = new List<Placanja>();

        //public virtual ICollection<Rezervacije> Rezervacijes { get; set; } = new List<Rezervacije>();
    }
}
